apps/deploy-web/src/hooks/useDeploymentMinimumEscrowAlertText.tsx

```diff
@@ -0,0 +1,24 @@
+import { useMemo, useState } from "react";
+
+import { useChainParam } from "@src/context/ChainParamProvider";
+import { useWallet } from "@src/context/WalletProvider";
+import { useDenomData } from "@src/hooks/useWalletBalance";
+
+export const useDeploymentMinimumEscrowAlertText = () => {
+  const { isManaged } = useWallet();
+  const [sdlDenom] = useState("uakt");
+  const depositData = useDenomData(sdlDenom);
+  const { minDeposit } = useChainParam();
+
+  return useMemo(() => {
+    return isManaged ? (
+      <>
+        To create a deployment, you need to have at least <b>${depositData?.min}</b> in an escrow account.{" "}
```

Comment:
I didn't think there would be rendering involved, but in this case I think it makes sense to make it a component instead of hook.